app/controllers/api/v2/events_controller.rb

```diff
@@ -2,7 +2,10 @@ class Api::V2::EventsController < Api::BaseController
 
   def create
     event = JSON.parse(request.body.read)
-    ap event
+    push_events = event['events'].find_all do |e|
```

Comment:
Sounds good to me. Should I write the `Event` class under the models' directory?